src/js/shield_defs.js

```diff
@@ -3038,42 +3025,87 @@ export function loadShields(shieldImages) {
+      refsByWayName: {
+        "Audubon Parkway": "AU",
+        "Bluegrass Parkway": "BG",
+        "Bluegrass Pkwy": "BG",
+        "Cumberland Parkway": "LN",
+        "Cumberland Pkwy": "LN",
+        "Hal Rogers Parkway": "HR",
+        "Hal Rogers Pkwy": "HR",
+        "Mountain Parkway": "MP",
+        "Mountain Pkwy": "MP",
+        "Purchase Parkway": "JC",
+        "Purchase Pkwy": "JC",
+        "Western Kentucky Parkway": "WK",
+        "Western Kentucky Pkwy": "WK",
+      },
+    },
+    shields["US:KY:AA"]
+  );
+
   shields["US:MI"].overrideByRef = {
     185: {
       backgroundImage: shieldImages.shield40_diamond_brown,
       textColor: Color.shields.white,
     },
   };
 
-  shields["US:PA:Allegheny:Belt"] = {
-    notext: true,
-    overrideByRef: {
-      "Red Belt": {
-        backgroundDraw: (ref) =>
-          ShieldDraw.paBelt(Color.shields.red, Color.shields.black),
-      },
-      "Orange Belt": {
-        backgroundDraw: (ref) =>
-          ShieldDraw.paBelt(Color.shields.orange, Color.shields.black),
-      },
-      "Yellow Belt": {
-        backgroundDraw: (ref) =>
-          ShieldDraw.paBelt(Color.shields.yellow, Color.shields.black),
-      },
-      "Green Belt": {
-        backgroundDraw: (ref) =>
-          ShieldDraw.paBelt(Color.shields.green, Color.shields.white),
+  shields["US:PA:Allegheny:Belt"].overrideByRef = {
+    "Red Belt": paBeltShield(Color.shields.red, Color.shields.black),
+    "Orange Belt": paBeltShield(Color.shields.orange, Color.shields.black),
+    "Yellow Belt": paBeltShield(Color.shields.yellow, Color.shields.black),
+    "Green Belt": paBeltShield(Color.shields.green, Color.shields.white),
+    "Blue Belt": paBeltShield(Color.shields.blue, Color.shields.white),
+    "Purple Belt": paBeltShield(Color.shields.purple, Color.shields.white),
+  };
+
+  shields["US:TX:Fort_Bend:FBCTRA"] = Object.assign(
+    {
+      refsByWayName: {
+        "Fort Bend Parkway Toll Road": "FBP",
+        "Fort Bend Westpark Tollway": "WPT",
       },
-      "Blue Belt": {
-        backgroundDraw: (ref) =>
-          ShieldDraw.paBelt(Color.shields.blue, Color.shields.white),
+    },
+    {
+      backgroundImage: shieldImages.shield40_us_tx_fbctra,
+      textColor: Color.shields.white,
+      padding: {
+        left: 3,
+        right: 3,
+        top: 2,
+        bottom: 8,
       },
-      "Purple Belt": {
-        backgroundDraw: (ref) =>
-          ShieldDraw.paBelt(Color.shields.purple, Color.shields.white),
+    }
+  );
+  shields["US:TX:Harris:HCTRA"] = Object.assign(
+    {
+      refsByWayName: {
+        "East Sam Houston Tollway North": "SHT",
+        "East Sam Houston Tollway South": "SHT",
+        "North Sam Houston Tollway East": "SHT",
+        "North Sam Houston Tollway West": "SHT",
+        "South Sam Houston Tollway East": "SHT",
+        "South Sam Houston Tollway West": "SHT",
+        "West Sam Houston Tollway North": "SHT",
+        "West Sam Houston Tollway South": "SHT",
```

Comment:
I look forward to eventually being able to use `short_name` on the relation instead of `name` on the way.